index.html.ejs

```diff
@@ -23,8 +23,8 @@
   <base href="<%= htmlWebpackPlugin.options.baseHref %>" />
   <% } %>
 
-  <link href="http://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
-  <link href="http://fonts.googleapis.com/css?family=Roboto:400,700,300" rel="stylesheet" type="text/css">
+  <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
```

Comment:
We should probably just use the https protocol since it's available (https://www.paulirish.com/2010/the-protocol-relative-url/).